### ISSUE TITLE
scatter is not changing color in Axes3D

### ISSUE THREAD

**Issue by orena1** [matplotlib/matplotlib#5974]
This simple example, should create an image with a single **red** circle

```
import matplotlib.pyplot as ploty

from mpl_toolkits.mplot3d import Axes3D

ploty.ion()

fig = ploty.figure()
ax = fig.add_subplot(111, projection='3d')
ax.scatter(0,0,0,color='red')
```

But this is what I get

![image](https://cloud.githubusercontent.com/assets/8983713/12887893/e3e5a452-ce7f-11e5-8f31-156d66c1bf42.png)

Is this a bug?

Thanks.

**Comment by WeatherGod** [comment]
Sigh, yeah, it is a bug. I have to check to see if it is still in effect
for v2.0. The work-around is to use "c" instead of "color".

**Comment by tacaswell** [comment]
So for `plot` you need to use `'color'` to avoid py3k dict issues and with scatter you need to use `'c'`?

**Comment by WeatherGod** [comment]
Ugh, I didn't even think about that. Only that 'c' is explicitly processed
by scatter3d to delete masked points, and 'color' is passed through. Both
'c' and 'color' would then show up to the 2d call of scatter().

**Comment by orena1** [comment]
By the way, facecolor is also not working, (edgecolor is working)

**Comment by WeatherGod** [comment]
For the purposes of scatter, facecolor and color are pretty much aliases,
which get clobbered by the 'c' value that scatter3d is emitting...

**Comment by soupault** [comment]
I agree, this is a really sad issue.
I'm using `(..., c=colors_vec, edgecolors='none', ...)` to get _expected default_ behaviour (no edge).
Are there any reasons for `scatter` kwargs not to be unified with other plotting functions?

**Comment by mdboom** [comment]
@WeatherGod: Do you think this is an easy fix?

**Comment by WeatherGod** [comment]
Might not fix everything, but adding kwarg normalization to the 3d
functions could probably fix some of this, I think?

**Comment by WeatherGod** [comment]
#6426 might actually be relevant here?

**Comment by WeatherGod** [comment]
So, essentially, I need to update the scatter3d function to not emit 'c' in some cases, I think?

**Comment by tacaswell** [comment]
Yes.

**Comment by WeatherGod** [comment]
I submitted PR #6439 that should fix this issue.

**Comment by WeatherGod** [comment]
Closed via #6439
